src/main/java/org/bahmni/reports/model/Report.java

```diff
@@ -52,7 +52,8 @@
         @JsonSubTypes.Type(value = ObsCannedReport.class, name = "obsCannedReport"),
         @JsonSubTypes.Type(value = GenericProgramReport.class, name = PROGRAMS),
         @JsonSubTypes.Type(value = AggregationReport.class, name = "aggregation"),
-        @JsonSubTypes.Type(value = ConcatenatedReport.class, name = "concatenated"),
+        @JsonSubTypes.Type(value = OpenmrsConcatenatedReport.class, name = "concatenated"),
```

Comment:
@sumanmaity112 am I correct that keeping `name = "concatenated"` for `OpenmrsConcatenatedReport` is what would ensure that this does not provoke any regressions?

Reply:
Hi @mks-d, yes, you are right. As the type **concatenated** is same, it will not affect any pre-existing report configuration. For release 0.93 we think how we can make bahmni report more generic

Reply:
In that case I don't have any objections to merge, @angshu do you agree?

Reply:
tested the changes locally. Works as expected. Merging the PR.